CONTRIBUTING.md

```diff
@@ -33,6 +33,9 @@ You can run PHP tests with `vendor/bin/phpunit` and JavaScript tests with `npm t
 
 If you need any help with this please don't hesitate to ask.
 
+> :warning: If your filesystem uses CRLF instead of LF you may run into some issues when running the tests
```

Comment:
I added this because I've been struggling for a good hour with this while running tests locally